### DISCUSSION TITLE
only dev-main.uf2 works

### DISCUSSION THREAD

**Discussion by sidewinder13** [dawigit/picoclock#4] [General]
I tried to recompile and send the new main.uf2
Freezes completely after 5 min or so, the same with every option like min or  editpos. What am i doing wrong. I searched an i searched but i cannot find anything.. getting  really frustrated ..

**Comment by dawigit** [comment]
Try building a 'release' instead of 'debug' binary.

**Reply by JWegener** [reply]
Can you provide more info about how to do the build? We do that before the flash? I get all sorts of errors about missing libraries trying to do the build

**Reply by dawigit** [reply]
Did you install the SDK?
Were you able to build?

**Reply by JWegener** [reply]
Eventually yes, but it was 3 months ago now and I forget what I did :) I maybe had to use an older version of the IDE that was still javabased.

**Comment by dawigit** [comment]
You build your own binary OR flash 'main.uf2' (./picoclock/uf2/main.uf2) - just as you wish.

The RP2040-LCD-1.28 is a RaspberryPi Pico with a display on it - simply spoken.
That's why you need to install the 'pico-sdk' first.
I use a RaspberryPi 4 or my Linux computer for development.
Check out RaspberryPi's documentation [https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf].
-> "2.1. Get the SDK and examples"
